package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pushwoosh-cordova-plugin",
-  "version": "7.13.0-MABS4-OS1",
+  "version": "7.17.1",
```

Comment:
Should keep the -MABS4-OS suffix

Reply:
I was planning to do it after merging, but if the point of this pull request is to upgrade and include our custom code it makes more sense to do it now 👍

Reply:
Yes, if the code is already customized in the development branch, the tag should contain `-MABS4-OS`